require.js

```diff
@@ -1119,12 +1149,46 @@
     };
 
 
+    Require.parseMJSONDependencies = function parseMJSONDependencies(jsonRoot) {
+
+        // Clear commented require calls
+        var rootEntries = Object.keys(jsonRoot),
+            i=0, iLabel, dependencies = [], iLabelObject;
+        while ((iLabel = rootEntries[i])) {
+            iLabelObject = jsonRoot[iLabel];
+            if(iLabelObject.hasOwnProperty("prototype")) {
+                dependencies.push(iLabelObject["prototype"]);
+            }
+            else if(iLabelObject.hasOwnProperty("object")) {
+                dependencies.push(iLabelObject["object"]);
+            }
+            i++;
+        }
+        return dependencies;
+    };
+
+
     // Built-in compiler/preprocessor "middleware":
 
     Require.DependenciesCompiler = function(config, compile) {
         return function(module) {
             if (!module.dependencies && module.text !== void 0) {
-                module.dependencies = config.parseDependencies(module.text);
+                if (module.location && (endsWith(module.location, ".meta") || endsWith(module.location, ".mjson"))) {
+
+                    if (typeof module.exports !== "object" && typeof module.text === "string") {
+                        module.parsedText = JSON.parse(module.text);
```

Comment:
What's the incentive in adding `parsedText` to the public module api?

Reply:
If somehow someone might want to process the pure JSON of a .mjson for some reasons

Reply:
Do we have a real use case in mind? It's supposed to be transparent, right? I would prefer not to add anything extra to the public module API unless we really need to.